secator/serializers/file.py

```
@@ -0,0 +1,50 @@
+
+	def on_cmd_done(self, runner):
+		# Find output paths in command output using regex
+		if self.output_path_regex:
+			matches = re.findall(self.output_path_regex, runner.output)
+			if not matches:
+				runner.add_result(Warning(message=f'Could not find output file from regex {self.output_path_regex}'), print=True)  # noqa: E501
+				return
+			self.output_path = matches
+
+		output_paths = self.output_path if isinstance(self.output_path, list) else [self.output_path]
+	for output_path in output_paths:
```

Ruff (0.11.9)

Undefined name `output_paths`

(F821)